language-server/package.json

```diff
@@ -119,6 +119,18 @@
           "default": {},
           "markdownDescription": "Enable features in languages that are not supported by default. Add a mapping here between the new language and an already supported language.\n E.g.: `{\"plaintext\": \"html\"}`"
         },
+        "tailwindCSS.files.exclude": {
```

Comment:
This one also seems like handled in the extension.

Reply:
Actually maybe not: https://github.com/tailwindlabs/tailwindcss-intellisense/blob/9bc82821c7dfef20db694f5fd255869b766c01f1/packages/tailwindcss-language-server/src/util/isExcluded.ts#L13

Though I see it also used in the extension. I wonder how much we are missing without the extension glue code.

Reply:
Yes, it seems like this setting is used both on the language server and in the VS code extension,
For now I would leave this setting.